examples/v1beta1/pipelineruns/pipelinerun-with-when-expressions.yaml

```diff
@@ -99,6 +100,18 @@ spec:
             - name: echo
               image: ubuntu
               script: exit 1
+      - name: task-should-be-executed-after-skipped-parent-task # whenSkipped set to runBranch in parent task
```

Comment:
Unfortunately, we do not have a way to verify that this task was executed. This looks good for now, but we might want to create an `e2e` test in future to verify such scenarios.

Reply:
yes, that'd be a great next step to this: https://github.com/tektoncd/pipeline/issues/3076